drivers/hmis_external_apis/extensions/hmis/hud/client_extension.rb

```diff
@@ -18,6 +18,18 @@ module ClientExtension
                   -> { where(namespace: HmisExternalApis::AcHmis::Mci::SYSTEM_ID) },
                   class_name: 'HmisExternalApis::ExternalId',
                   as: :source
+
+          # Used by ClientSearch concern
+          def self.injected_search(where, text, numeric:, alpha_numeric:, **_kwargs)
+            return where unless numeric || alpha_numeric
```

Comment:
It doesn't look like you are using numeric or alpha_numeric, why check for them?

Reply:
It just doesn't really make sense to do the External ID search unless the string is alphanumeric. The check of `numeric` is redundant since `alpha_numeric` includes numeric. I'll move the check to client search though since this will now be `search_by_external_id`